netbsd/vm/setup.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+VERSION="9.99.99"
+ARCH="amd64"
+
+IMAGE_FILENAME="NetBSD-$VERSION-$ARCH.iso"
+
+function delete_confirmation() {
+    if [ ! -f disk.qcow2 ]; then
+        return
+    else
+        echo "Setup utility will permanently delete the previous disk image!"
+        while true; do
+            read -p "Do you want to proceed? y/n " yn
+            case $yn in
+                [Yy]* ) rm disk.qcow2; break;;
+                [Nn]* ) exit 1;;
+                * ) echo "Please answer yes or no.";;
+            esac
+        done
+    fi 
+}
+
+function safety_checks() {
+    if [ -f disk.qcow2 ]; then
+        echo "disk.qcow2 already exists, won't format! Remove it to do setup."
+        exit 1
+    fi
+
+    if [ ! -f $IMAGE_FILENAME ]; then
+        echo "Installer image .iso not found, copy it or create a symlink here"
+        exit 1
+    fi
+}
+
+function dependency_checks() {
+    local fulfilled=y
+
+    if [ ! -f OVMF.fd ]; then
+        echo 'OVMF.fd is missing!'
+        fulfilled=n
+    fi
+
+    if ! command -v python3 > /dev/null; then
+        echo 'python3 not found!'
+        fulfilled=n
+    fi
+
+    if ! command -v rsync > /dev/null; then
```

Comment:
`rsync` is used, but result of running it isn't. Looks like this and `prepare_mirror()` can both be removed.

Reply:
done